infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -71,16 +71,60 @@ type FeatureStoreServices struct {
 // OfflineStore configures the deployed offline store service
 type OfflineStore struct {
 	ServiceConfigs `json:",inline"`
+	// +optional
```

Comment:
the `// +optional` comments aren't necessary as it is equivalent to using`omitempty` in the json field descrip. imo, let's pick one approach and stick w/ that.